tests/visco_plastic_peierls_cutoff_test/screen-output

```diff
@@ -1,11 +1,15 @@
+-----------------------------------------------------------------------------
```

Comment:
These lines can be removed from the test outputs

Reply:
Yeah, you are right, I have removed them.